db/migrate/20170216193758_add_vm_table_comments.rb

```diff
@@ -0,0 +1,81 @@
+class AddVmTableComments < ActiveRecord::Migration[5.0]
+  def change
+    change_table_comment :vms, "Virtual Machines and Templates"
+
+    change_column_comment :vms, :autostart, "A boolean value that indicates whether or not the VM is set to autostart."
+    change_column_comment :vms, :availability_zone_id, "Availability zone that the VM is part of. FK to the availability_zones table."
+    change_column_comment :vms, :busy, "HELP"
+    change_column_comment :vms, :boot_time, "The last time the VM was started."
+    change_column_comment :vms, :cloud, "A boolean indicating whether or not the VM/Template is cloud or infra."
+    change_column_comment :vms, :cloud_network_id, "Cloud (virtual) network the VM/Template is associated with. FK to the cloud_networks table."
+    change_column_comment :vms, :cloud_subnet_id, "Subnet associated with the VM. FK to the cloud_subnets table."
+    change_column_comment :vms, :cloud_tenant_id, "Cloud tenant associated with the VM. FK to the cloud_tenants table."
+    change_column_comment :vms, :config_xml, "HELP"
```

Comment:
This was originally intended to store the configuration of the VM from the provider.  It does not look like this field is used anymore.